modelci/hub/__init__.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: Li Yuanming
+Email: yli056@e.ntu.edu.sg
+Date: 9/19/2020
+"""
+__all__ = ['converter', 'manager', 'client', 'utils', 'Profiler']
+
+from . import client
+from . import converter
+from . import manager
+from . import utils
+from .profiler import Profiler
```

Comment:
as profiler?

all the others are lowercase

Reply:
because `Profiler` is a class, others are modules.